### DISCUSSION TITLE
marl_env

### DISCUSSION THREAD

**Discussion by EhsanSabouni** [metadriverse/metadrive#728] [General]
Hi everyone

I have a query. I am interested in creating a marl_env for ramp followed by a straight road segment. Is there any straightforward way so I can create this environment. I only see intersection and roundabouts and a few other environments for multi-agent reinforcement application.

Thank you in advance.

**Comment by QuanyiLi** [comment]
Hi @EhsanSabouni,

Just take a look at how we create MARL maps, you only need to connect two blocks to get what you want:
https://github.com/metadriverse/metadrive/blob/05ec6c8ea9a9fc77742b8af9967e63b43b4abaae/metadrive/envs/marl_envs/marl_tollgate.py#L113
